fe/fe-core/src/main/java/org/apache/doris/statistics/util/StatisticsUtil.java

```diff
@@ -657,7 +656,7 @@ public static long getRowCountFromFileList(HMSExternalTable table) {
             estimatedRowSize += column.getDataType().getSlotSize();
         }
         if (estimatedRowSize == 0) {
-            return 1;
+            return 0;
```

Comment:
For `getRowCountFromFileList()`, I think we need to check if this is a hive table or not.
Only hive table support get row count from file list.

Reply:
Will fix this